src/applications/mhv/medical-records/components/LabsAndTests/ChemHemDetails.jsx

```diff
@@ -122,11 +127,39 @@ const ChemHemDetails = props => {
       })),
     };
 
-    makePdf(
-      'microbiology_report',
-      scaffold,
-      'Microbiology details',
-      runningUnitTest,
+    makePdf('chem/hem_report', scaffold, 'Chem/Hem details', runningUnitTest);
+  };
+
+  const generateChemHemTxt = async () => {
+    const content = `\n
+${record.name}\n
+Date entered: ${record.date}\n
+Record Details:\n
+_____________________________________________________\n\n
+Type of test: ${record.reason} \n
+Sample tested: ${record.clinicalHistory} \n
+Ordered by: ${record.orderedBy} \n
+Order location: ${record.orderingLocation} \n
+Collecting location: ${record.imagingLocation} \n
+Provider notes: ${processList(record.comments)} \n;
+_____________________________________________________\n\n
+Results: \n
+  ${record.results.map(
```

Comment:
Because this is a string literal, these two spaces will be printed in your final output. Left align your "${..."

Reply:
![image](https://github.com/department-of-veterans-affairs/vets-website/assets/9909777/4bdb160c-ad97-49e6-bbad-d4d760f81934)